src/components/surveys/survey-design/question-edit/QuestionEditRhs.tsx

```diff
@@ -119,7 +119,7 @@ const QuestionEditRhs: FunctionComponent<QuestionEditProps> = ({step, onChange,
             <StyledLabel14 htmlFor="q_id">Question Identifier</StyledLabel14>
             <StyledSimpleTextInput
               sx={{display: 'block', width: '200px'}}
-              onChange={e => onChange({...step, identifier: e.target.value})}
+              $readOnly={true}
```

Comment:
Make the step identifier "read only", instead of disabled:

https://github.com/BridgeDigitalHealth/open-bridge-web/assets/26949006/9e6b7cfa-6052-481c-9091-36e5a24d75a7